packages/jest-runtime/src/index.ts

```diff
@@ -322,14 +320,6 @@ class Runtime {
     });
   }
 
-  static runCLI(args?: Config.Argv, info?: Array<string>): Promise<void> {
```

Comment:
should we keep these and just throw an error saying the function have moved to `jest-repl`?

Reply:
Great idea, i'll add this.